### PR TITLE
List view controller

### PR DESCRIPTION
**Pull request by khrystyna-sb**
Screenshot 2021-12-09 at 12 24 21

### CODE REVIEW

**Review by Globalika — CHANGES_REQUESTED** [review]
miss click

File: Countries/FirstScreen/CountryTableViewCell.swift [hunk 1]
```diff
@@ -52,18 +61,28 @@ class CountryTableViewCell: UITableViewCell {
...
+        label.lineBreakMode = .byTruncatingTail
         return label
     }()
     
     lazy private var regionLabel: UILabel = {
         let label = UILabel()
         label.textAlignment = .left
+        label.numberOfLines = 0
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = UIFont.systemFont(ofSize: Constants.fontSize)
```
Comment (Globalika) [inline Countries/FirstScreen/CountryTableViewCell.swift]: why capitalLabel have lineBreakMode and others dont? or vise versa

File: Countries/FirstScreen/ListViewController.swift [hunk 2]
```diff
@@ -11,24 +11,38 @@ class ListViewController: UITableViewController {
...
     
     var countries: [CountriesApiQuery.Data.Country] = []
     
+//    var navigationBarTitleView: UIView = {
+//        let view = UIView()
+//        view.backgroundColor = UIColor(red: 0.839, green: 0.761, blue: 0.553, alpha: 0.5)
+//        view.translatesAutoresizingMaskIntoConstraints = false
+//        return view
+//    }()
+
```
Comment (Globalika) [inline Countries/FirstScreen/ListViewController.swift]: do you need theese comment lines?

File: Countries/FirstScreen/ListViewController.swift [hunk 3]
```diff
@@ -11,24 +11,38 @@ class ListViewController: UITableViewController {
...
+        setupNavigationItem()
         registerTableView()
         loadData()
     }
     
+    func setupNavigationItem() {
+        self.navigationItem.title = "Country list"
+//        navigationController?.navigationBar.barTintColor = UIColor.green
+//        self.navigationItem.titleView = navigationBarTitleView
```
Comment (Globalika) [inline Countries/FirstScreen/ListViewController.swift]: saa

File: Countries/Headers/CountryTableViewHeader.swift [hunk 4]
```diff
@@ -12,32 +12,33 @@ class CountryTableViewHeader: UITableViewHeaderFooterView {
     
     private enum Constants {
         static let earthImageName = "earth"
-        static let labelText = "Choose a card :)"
         static let backgroundColor = UIColor(red: 0.945, green: 0.945, blue: 0.945, alpha: 1)
         static let fontSize: CGFloat = 44
+        static let labelText = "Choose a card"
```
Comment (Globalika) [inline Countries/Headers/CountryTableViewHeader.swift]: where do the smile go? :(

File: Countries/FirstScreen/CountryTableViewCell.swift [hunk 5]
```diff
@@ -112,22 +126,22 @@ class CountryTableViewCell: UITableViewCell {
...
         ])
     }
     
     public func configure(counrty: CountriesApiQuery.Data.Country) {
-        nameLabel.text = counrty.name
-        capitalLabel.text = counrty.capital
-        regionLabel.text = counrty.continent.name
+        nameLabel.text = "country\n" + counrty.name.uppercased()
+        capitalLabel.text = "capital\n" + (counrty.capital?.uppercased() ?? "NA")
+        regionLabel.text = "region\n" + counrty.continent.name.uppercased()
```
Comment (Globalika) [inline Countries/FirstScreen/CountryTableViewCell.swift]: NA should be in constants. cant be sure about "country\n" and so on

**Review by Globalika — CHANGES_REQUESTED** [review]
i left you comments. please resolve it

File: Countries/FirstScreen/CountryTableViewCell.swift (hunk 1 above)
Comment (khrystyna-sb) [reply Countries/FirstScreen/CountryTableViewCell.swift]: thank you, I just tested it and forgot to delete

**Review by Globalika — APPROVED** [review]

**Review by apolwork — APPROVED** [review]

**Pull request merged**